python-clusters/attach-gke-cluster/cluster.py

```diff
@@ -30,7 +30,8 @@ def start(self):
         # delegate the creation of the kube config file to gcloud to use the client go auth plugin
         kube_config_path = os.path.join(os.getcwd(), 'kube_config')
         region_or_zone = clusters.region if is_regional else clusters.zone
-        create_kube_config_file(clusters.project_id, cluster.name, is_regional, region_or_zone, kube_config_path)
+        is_dns_endpont = self.config.get('isDnsEndpoint', False)
+        create_kube_config_file(clusters.project_id, cluster.name, is_regional, region_or_zone, kube_config_pa, is_dns_endpoint)
```

Comment:
The `kube_config_path` variable name got truncated
```suggestion
        create_kube_config_file(clusters.project_id, cluster.name, is_regional, region_or_zone, kube_config_path, is_dns_endpoint)
```